modules/statics/src/coins.ts

```diff
@@ -1247,8 +1247,8 @@ export const coins = CoinMap.fromCoins([
     UnderlyingAsset.ARBETH,
     CoinKind.CRYPTO
   ),
-  ofc('8b93e788-52fa-4fd6-b499-40f13fe194fc', 'ofcCoreum', 'Coreum', 6, UnderlyingAsset.COREUM, CoinKind.CRYPTO),
-  tofc('3091d79a-7737-4493-b3e6-6765998b9486', 'ofctCoreum', 'Test Coreum', 6, UnderlyingAsset.COREUM, CoinKind.CRYPTO),
+  ofc('8b93e788-52fa-4fd6-b499-40f13fe194fc', 'ofccoreum', 'Coreum', 6, UnderlyingAsset.COREUM, CoinKind.CRYPTO),
+  tofc('3091d79a-7737-4493-b3e6-6765998b9486', 'ofctcoreum', 'Test Coreum', 6, UnderlyingAsset.COREUM, CoinKind.CRYPTO),
```

Comment:
it would be better if you add tofc to the end of the tofc group. Generally all the additions happens at the end of the list.

Reply:
you can follow how tarbeth is added.

Reply:
its fine now right?

Reply:
it looks fine now ?

Reply:
yeah this works.